cns/restserver/ipam.go

```diff
@@ -152,9 +152,12 @@ func (service *HTTPRestService) MarkIpsAsAvailableUntransacted(ncID string, newH
 	if ncInfo, exist := service.state.ContainerStatus[ncID]; !exist {
 		logger.Errorf("Can't find NC with ID %s in service state, stop updating its pending programming IP status", ncID)
 	} else {
-		previousHostNCVersion := ncInfo.HostVersion
+		previousHostNCVersion, err := strconv.Atoi(ncInfo.HostVersion)
+		if err != nil {
+			logger.Printf("[MarkIpsAsAvailableUntransacted] Get int value from ncInfo.HostVersion %s failed for %v", ncInfo.HostVersion, err)
```

Comment:
nit: better to use colon than `for`
```
[MarkIpsAsAvailableUntransacted] Get int value from ncInfo.HostVersion %s failed: %v
```
also, if it fails, the rest of the block should not execute, since it would be acting on `previousHostNCVersion` being `0`. Probably should `continue` here

Reply:
I'll return here.